jsplot/src/main/java/tech/tablesaw/plotly/components/Marker.java

```diff
@@ -129,7 +135,9 @@ public String asJavascript() {
     Map<String, Object> context = new HashMap<>();
     context.put("size", size.length == 1 ? size[0] : Utils.dataAsString(size));
     if (colorScalePalette != null) {
```

Comment:
If colorScalePalette always takes precedence over colorScaleCustomRange should we mention that in the javadoc somewhere?